extra/resources/SysInfo.in

```diff
@@ -354,7 +354,13 @@ SysInfo_reload_agent() {
 }
 
 SysInfo_validate() {
-    return $OCF_SUCCESS
+    # If specified, is min_disk_free a non-negative integer followed by a disk unit?
+
+    pat="[1-9][0-9]*[K|B|K|M|G|T|P]"
```

Comment:
K is in there twice, and we should accept lowercase as well. We also need to accept a bare number without a unit.

For regular expressions, bracketed characters don't need a pipe between them, e.g. [BKMGTP]